share/pot/clone.sh

```diff
@@ -313,6 +324,33 @@ pot-clone()
 					${EXIT} 1
 				fi
 				;;
+			d)
+				case $OPTARG in
+					"inherit")
+						_dns="inherit"
+						;;
+					"pot")
+						_dns="pot"
+						;;
+					"off")
+						_dns="off"
+						;;
+					custom:*)
+						if [ -r "${OPTARG##custom:}" ]; then
+							_dns=$OPTARG
+						else
+							_error "The file ${OPTARG##custom:} is not valid or reabale"
+							clone-help
+							${EXIT} 1
+						fi
+						;;
+					*)
+							_error "The dns ${OPTARG} is not valid option"
+							clone-help
```

Comment:
General remark: It would be good if showing the help screen as a result of a syntax problem (= error condition) would go to stderr, while requesting it using `-h` goes to stdout.

Could be accomplished, e.g., by calling

    >&2 clone-help

or by writing a wrapper in common.sh

    exit-error()
    {
             >&2 cat
             ${EXIT} "$1"
    }

    clone-help | exit-error 1

In general, error messages should go to stderr.

Material for a separate issue/discussion.

Reply:
yes, you are correct, but it doesn't makes sense to fix it in this PR or for on command only.
For sure, material for a separate issue